delivery_admin/views/react/orders.jsx

```diff
@@ -18,11 +18,50 @@ class Orders extends React.Component {
     })
   }
 
+  count_orders(orders){
+    var count = {}
+    for(var i in orders){
```

Comment:
dont use var. if the variable does not change use const, if it does use let . The reason is that [the scope of var is a lot broader](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let) and this causes bugs in the long run